weave-daemonset.yaml

```diff
@@ -55,6 +56,11 @@ spec:
               cpu: 25m
           securityContext:
             privileged: true
+        - name: weave-npc
+          image: weaveworks/weave-npc:latest
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
```

Comment:
Suggest adding a `resources` section, otherwise it is assumed to need 100 mCPU and will not start on some configurations.